guides/plugins/plugins/framework/extension/index.md

```diff
@@ -0,0 +1,23 @@
+---
+nav:
+  title: Extension Points
+  position: 40
+
+---
+
+# Extension Points
+
+Extension Points allow you to **replace core functionality** by intercepting and modifying the execution flow of system processes, unlike traditional events which are only for notifications.
+
+## Example
```

Comment:
this example looks a bit off here. If I understand that correctly this page is the entry point for the category?